.gitignore

```diff
@@ -1,8 +1,2 @@
-pkg
-bin
-hound.sublime-workspace
-data
 .vagrant
-pub/index.html
-pub/excluded_files.html
-config.json
```

Comment:
Ah I see, I just figured that since `make clean` deleted `ui/bindata.go` we wouldn't want to commit it, but that makes sense that if people aren't editing Hound and just want to run it they would need it. My bad.